src/test/java/org/cactoos/scalar/ConstantTest.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.scalar;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link Constant}.
+ * @author Elena Ihde-Simon (elena.ihde-simon@posteo.de)
+ * @version $Id$
+ * @since 0.30
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class ConstantTest {
+
+    @Test
+    public void returnsGivenValue() throws Exception {
+        MatcherAssert.assertThat(
+            "Can't return given value",
+            new Constant<>("Hello World").value(),
+            Matchers.endsWith("World")
```

Comment:
@aliceice Why not ``Matchers.equalTo("Hello World")`` here? ``endsWith`` does not guarantee that Constant doesn't change the encapsulated String somehow.

Reply:
The problem here is qulice. It does not allow to use the same String twice in a class. So either I define a variable beforehand and use this in both construction and assertion, or do it like this.

What do you suggest?

Reply:
@aliceice Right, I forgot how strict Qulice is... then I would say declare a variable beforehand :)